varnish/tox.ini

```diff
@@ -15,25 +13,13 @@ deps =
 passenv =
     DOCKER*
     COMPOSE*
+setenv =
+    varnish4: VARNISH_VERSION=4.1.7
```

Comment:
mind changing these to the actual versions?

Reply:
you mean the env factor? Like `varnish4.1.7: VARNISH_VERSION=4.1.7`?

Reply:
or just 

4.1.7: VARNISH_VERSION=4.1.7

Reply:
Done, looks cleaner 👍